From c914a1a701391c565203b10603ad0fbc52438bf7 Mon Sep 17 00:00:00 2001
From: Nathan Lynch <nathanl@linux.ibm.com>
Date: Mon, 25 Sep 2023 11:41:22 -0500
Subject: [PATCH 4/6] librtas: vendor papr-miscdev.h

This is a common header used by other kernel uapi headers that we will
copy into the source tree.

Signed-off-by: Nathan Lynch <nathanl@linux.ibm.com>
Signed-off-by: Tyrel Datwyler <tyreld@linux.ibm.com>
---
 Makefile.am                | 4 +++-
 librtas_src/papr-miscdev.h | 9 +++++++++
 2 files changed, 12 insertions(+), 1 deletion(-)
 create mode 100644 librtas_src/papr-miscdev.h

diff --git a/Makefile.am b/Makefile.am
index 1385ac7..d0cabfb 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -35,7 +35,9 @@ librtas_la_SOURCES = \
 	librtas_src/sysparm.c
 
 library_include_HEADERS += librtas_src/librtas.h
-noinst_HEADERS += librtas_src/internal.h
+noinst_HEADERS += \
+	librtas_src/internal.h \
+	librtas_src/papr-miscdev.h
 
 # See "Updating library version information" in the libtool manual for
 # how to maintain these values. They are *not* tied to the release
diff --git a/librtas_src/papr-miscdev.h b/librtas_src/papr-miscdev.h
new file mode 100644
index 0000000..c0b1072
--- /dev/null
+++ b/librtas_src/papr-miscdev.h
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note
+#ifndef _UAPI_PAPR_MISCDEV_H_
+#define _UAPI_PAPR_MISCDEV_H_
+
+enum {
+	PAPR_MISCDEV_IOC_ID = 0xb2,
+};
+
+#endif /* _UAPI_PAPR_MISCDEV_H_ */
-- 
2.43.0

